src/commands/remove-mst.ts

```diff
@@ -0,0 +1,120 @@
+import { GluegunToolbox, filesystem } from "gluegun"
+import * as pathlib from "path"
+import { boolFlag } from "../tools/flag"
+import { p } from "../tools/pretty"
+import { packager } from "../tools/packager"
+
+const { run: jscodeshift } = require("jscodeshift/src/Runner")
+
+export const mstDependenciesToRemove = [
+  "mobx",
+  "mobx-react-lite",
+  "mobx-state-tree",
+  "reactotron-mst",
+]
+
+module.exports = {
+  alias: ["rmst", "remove-mst"],
+  description:
+    "Remove MobX-State-Tree code from generated boilerplate. Add --dry-run to see what would be removed.",
+  run: async (toolbox: GluegunToolbox) => {
+    const { parameters } = toolbox
+
+    const CWD = process.cwd()
+    const TARGET_DIR = parameters.first ?? CWD
+    const dryRun = boolFlag(parameters.options.dryRun) ?? false
+
+    // TODO: full dryRun support
+
+    p()
+    p(`Removing MobX-State-Tree code from '${TARGET_DIR}'${dryRun ? " (dry run)" : ""}`)
```

Comment:
What about a try/catch here and if running into any issues, let's warn the user the automated removal could not be completed and refer them to the recipe to do so manually?

Reply:
A suggestion: if we want to refer people to document officially, I think that document should be docs in Ignite, and not a cookbook recipe.

Reply:
I added a link to the recipe in the existing try/catch as part of the `new` flow 👍 
I think since it's a tutorial and intended as a backup if this fails, it makes sense to keep it in Cookbook, IMO. 
<img width="1507" alt="image" src="https://github.com/user-attachments/assets/f3e3f143-8d9d-4da6-a154-5b795fffe75e">

Reply:
The main reason I say documentation and not a cookbook recipe was that the original intention of cookbook was it to have less obligation to keep it up-to-date (and therefore encourage contributions).

Linking to things officially from here makes it seem more like documentation, to me, which also implies it should continue to be tested with Ignite updates and stay accurate. Drawing a clear line on what we want to keep up-to-date and what's "this worked at one point" seems valuable to me.

Medium GAFO on this, definitely open to disagreement, but that's the thinking behind it.

Reply:
I'm going to leave as a Cookbook link for now for simplicity, but if we find that the cookbook recipe falls out of date or if we want to integrate the MobX Removal instructions into official Ignite docs in the future, we can make those changes separately